src/lib/api/api.go

```diff
@@ -505,26 +524,27 @@ func (a *API) UpdateRoute(w http.ResponseWriter, r *http.Request) {
 			break
 		}
 	}
-	if _, err := a.config.Write(); err != nil {
```

Comment:
I have removed this since the ResponseStatus is going to be overwritten. Please let me know if we should use `return` here inside the conditional block.